code/modules/atmospherics/machinery/components/electrolyzer/electrolyzer.dm

```diff
@@ -166,6 +186,21 @@
 		return
 	return ..()
 
+/obj/machinery/electrolyzer/attack_hand_secondary(mob/user, list/modifiers)
+	if(!can_interact(user))
+		return
```

Comment:
```suggestion
		return SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN
```

Reply:
The other one needs this too, it's a runtime if _secondary doesn't return something.